web-console/src/dialogs/lookup-edit-dialog/lookup-edit-dialog.tsx

```diff
@@ -93,8 +194,365 @@ export const LookupEditDialog = React.memo(function LookupEditDialog(props: Look
+    },
+    {
+      name: 'extractionNamespace.namespaceParseSpec.valueFieldName',
+      type: 'string',
+      label: 'Value field name',
+      placeholder: `value`,
+      defined: (model: LookupSpec) =>
+        model.type === 'cachedNamespace' &&
+        !!model.extractionNamespace &&
+        model.extractionNamespace.type === 'uri' &&
+        model.extractionNamespace.namespaceParseSpec &&
+        model.extractionNamespace.namespaceParseSpec.format === 'customJson',
+    },
+    {
+      name: 'extractionNamespace.namespace',
+      type: 'string',
+      label: 'Namespace',
+      placeholder: 'some_lookup',
+      info: (
+        <p>
+          The namespace value in the SQL query:
+          <br />
+          SELECT keyColumn, valueColumn, tsColumn? FROM <strong>namespace</strong>.table WHERE
+          filter
+        </p>
+      ),
+      defined: (model: LookupSpec) =>
+        model.type === 'cachedNamespace' &&
+        !!model.extractionNamespace &&
+        model.extractionNamespace.type === 'jdbc',
+    },
+    {
+      name: 'extractionNamespace.connectorConfig.createTables',
+      type: 'boolean',
+      label: 'CreateTables',
+      info: 'Defines the connectURI value on the The connector config to used',
+      defined: (model: LookupSpec) =>
+        model.type === 'cachedNamespace' &&
+        !!model.extractionNamespace &&
+        model.extractionNamespace.type === 'jdbc',
+    },
+    {
+      name: 'extractionNamespace.connectorConfig.connectURI',
+      type: 'string',
+      label: 'ConnectURI',
+      info: 'Defines the connectURI value on the The connector config to used',
+      defined: (model: LookupSpec) =>
+        model.type === 'cachedNamespace' &&
+        !!model.extractionNamespace &&
+        model.extractionNamespace.type === 'jdbc',
+    },
+    {
+      name: 'extractionNamespace.connectorConfig.user',
+      type: 'string',
+      label: 'User',
+      info: 'Defines the user too be used by the connector config',
+      defined: (model: LookupSpec) =>
+        model.type === 'cachedNamespace' &&
+        !!model.extractionNamespace &&
+        model.extractionNamespace.type === 'jdbc',
+    },
+    {
+      name: 'extractionNamespace.connectorConfig.password',
+      type: 'string',
+      label: 'Password',
+      info: 'Defines the password too be used by the connector config',
+      defined: (model: LookupSpec) =>
+        model.type === 'cachedNamespace' &&
+        !!model.extractionNamespace &&
+        model.extractionNamespace.type === 'jdbc',
+    },
+    {
+      name: 'extractionNamespace.table',
+      type: 'string',
+      label: 'Table',
+      placeholder: 'some_lookup_table',
+      info: (
+        <p>
+          The table which contains the key value pairs. This will become the table value in the SQL
+          query:
+          <br />
+          SELECT keyColumn, valueColumn, tsColumn? FROM namespace.<strong>table</strong> WHERE
+          filter
+        </p>
```

Comment:
instead of `<br>` it is better to use multiple `<p>`s like so:

```
<>
        <p>
          The table which contains the key value pairs. This will become the table value in the SQL
          query:
          </p>
          <p>
          SELECT keyColumn, valueColumn, tsColumn? FROM namespace.<strong>table</strong> WHERE
          filter
        </p>
</>
```